<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Statistic</class>
 <widget class="QWidget" name="Statistic">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>99</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Statistic</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="1">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>起始日期</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QDateEdit" name="dateEdit">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="8">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="6">
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="2">
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>分类支出</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>账户支出</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>分类收入</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>账户收入</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>负债分布</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>图表类型：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>结束日期</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="5">
      <widget class="QDateEdit" name="dateEdit_2">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="9">
      <widget class="QPushButton" name="pushButton_back">
       <property name="text">
        <string>返回</string>
       </property>
      </widget>
     </item>
     <item row="1" column="7">
      <widget class="QPushButton" name="pushButton_generate">
       <property name="text">
        <string>生成统计图</string>
       </property>
      </widget>
     </item>
     <item row="0" column="7">
      <widget class="QPushButton" name="pushButton_account">
       <property name="text">
        <string>账户现有资产分布图</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QChartView" name="widget" native="true"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QWidget</extends>
   <header location="global">qchartview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>comboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>Statistic</receiver>
   <slot>on_comboBox_currentIdexChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>195</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>89</x>
     <y>35</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_comboBox_currentIdexChanged()</slot>
 </slots>
</ui>
